content/en/blog/falco-0-40.0/index.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Introducing Falco 0.40.0
+date: 2025-01-28
+author: Aldo Lacuku, Federico Di Pierro
+slug: falco-0-40-0
+tags: ["Falco","Release"]
+---
+
+Dear Falco Community, today we are happy to announce the release of Falco 0.40.0!
+
+This version brings several new features, performance improvements, and bug fixes that streamline Falco’s detection capabilities.
+During this release cycle, we merged more than 52 PRs on Falco and more than 150 PRs for libs and drivers, version 0.
+20.0 and version 8.0.0 respectively. Thank you to our maintainers and contributors, as this would not happen without your support and dedication!
+
+To learn all about these changes, read on!
+
+## What’s new? TL;DR
+
+*Key features:*
+
+* [Streamlined Falco docker images](#streamlined-falco-docker-images);
+* Falco static build has been reintroduced for `x86_64` binary using `musl`;
+* New [process filters](#introducing-new-processs-filters) allow to filter events based on process metadata;
+* Added support for `sendmmsg` and `recvmmsg` syscalls parameters;
+* [Plugins suggested output fields](#plugins-suggested-output-fields) are now available in the Falco engine [#3388](https://github.com/falcosecurity/falco/pull/3388), [#2116](https://github.com/falcosecurity/libs/pull/2116);
+
+This release also comes with [breaking changes](#breaking-changes-and-deprecations-) that you should be aware of before upgrading.
+
+## Major features and improvements
+
+The 0.40.0 release contains a number of feature and UX improvements, here are list of some of the key new capabilities.
+
+### Streamlined Falco docker images
```

Comment:
@alacuku @FedeDP 

We also introduced a breaking change here. Now the main `falcosecurity/falco` image switched from a Debian base image to a distroless one. 

Furthermore, the `falcosecurity/falco-no-driver` is deprecated and the experimental `falcosecurity/falco-distroless` does not exist anymore, since it became the main image.

This is very relevant and should be advertised.

Reply:
Done!